src/test/daml/Daml/Finance/Instrument/Generic/Test/CallableBond.daml

```diff
@@ -116,6 +117,7 @@ run = script do
       electionTime = dateToDateClockTime intermediateDate
       holdingCid = investorGenericTransferableCid
       amount = 5000.0
+      label = "Election - " <> show intermediateDate
```

Comment:
It probably makes sense to track it with an issue: we probably don't want the label to be provided by the investor but rather by the lifecycler.

Reply:
Done - https://github.com/DACH-NY/daml-finance/issues/304